reference/forms/types/collection.rst

```diff
@@ -324,7 +324,7 @@ type::
 entry_type
 ~~~~~~~~~~
 
-**type**: ``string`` **default**: ``Symfony\\Component\\Form\\Extension\\Core\\Type\\TextType``
+**type**: ``string`` **default**: ``Symfony\Component\Form\Extension\Core\Type\TextType``
```

Comment:
... then we should here add `'` before and after too, right? @javiereguiluz ?

Reply:
Yes, I added the `'` in both cases because they are strings.